CMakeLists.txt

```diff
@@ -241,6 +279,37 @@ if(NOT NONET)
   target_link_libraries(devilutionx PRIVATE sodium)
 endif()
 
+if(USE_SDL1)
+  target_link_libraries(devilutionx PRIVATE
+    ${SDL_TTF_LIBRARY}
+    ${SDL_MIXER_LIBRARY}
+    ${SDL_MIXER_LIBRARIES}
+    ${SDL_LIBRARY}
+    ${FREETYPE_LIBRARIES}
+    ${PNG_LIBRARIES})
+  if(WARPOS)
+    target_link_libraries(devilutionx PRIVATE
+      -lmikmod
+      -lvorbisfile
+      -lvorbisenc
+      -lvorbis
+      -logg
+      -lFLAC
+      -lbz2)
+  endif()
+else()
+  target_link_libraries(devilutionx PRIVATE
+    SDL2::SDL2main
+    ${SDL2_ttf_LIBRARY}
+    ${SDL2_mixer_LIBRARY}
+    ${FREETYPE_LIBRARIES}
+    ${PNG_LIBRARIES})

```

Comment:
PNG not needed

Reply:
Freetype needs it.

Reply:
@AJenbo Remember, we had this discussion about Amiga needing static libraries.

Reply:
I'm pretty sure you can disable the functionality that needs PNG support?

Reply:
Yes, if you recompile all libs, of course. The distributed libs has it enabled however.

Reply:
https://github.com/SDL-mirror/SDL_ttf/blob/b79d88f7b52c22f13dac3902e9b3b79438384266/external/freetype-2.9.1/CMakeLists.txt#L70

Reply:
I thought it was necessary to compile all the libs, are you saying I could have saved all that time rewriting the build script :D